www-client/links/links-2.20.1.recipe

```diff
@@ -4,12 +4,12 @@ HOMEPAGE="http://links.twibright.com/"
 COPYRIGHT="1999-2019 Mikulas Patocka
 	2000-2011 Petr Kulhavy, Karel Kulhavy, Martin Pergel"
 LICENSE="GNU GPL v2"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="http://links.twibright.com/download/links-$portVersion.tar.gz"
```

Comment:
```suggestion
SOURCE_URI="http://links.twibright.com/download/links-$portVersion.tar.bz2"
```

We should switch to the bz2 archive as it's smaller.

Reply:
Fixed.